moveos/moveos-stdlib/moveos-stdlib/sources/raw_table.move

```diff
@@ -110,7 +110,7 @@ module moveos_std::raw_table {
 
     native fun remove_box<K: copy + drop, V, B>(table_handle: address, key: K): Box<V>;
 
-    native fun destroy_empty_box(table_handle: address);
+    native fun destroy_empty_box<K>(table_handle: address);
 
-    native fun drop_unchecked_box(table_handle: address);
+    native fun drop_unchecked_box<K>(table_handle: address);
```

Comment:
I think `destroy_empty_box` and `drop_unchecked_box` do not require the `K` Type. You need to remove the ty_args length check: `assert_eq!(ty_args.len(), 1)`

https://github.com/rooch-network/rooch/blob/5ac0b5b9b534952f20059869a3da1d1c76dea42c/moveos/moveos-stdlib/src/natives/moveos_stdlib/raw_table/mod.rs#L547-L566

Reply:
has remove the `K` type for destroy_empty_box and drop_unchecked_box